### DISCUSSION TITLE
https://www.dcm4che.org/maven2/ unavailable

### DISCUSSION THREAD

**Discussion by slawomirkania** [dcm4che/dcm4che#917] [Q&A] [answered]
hello,

https://www.dcm4che.org/maven2/ repository is unavailable, anybody knows when it is going to be alive again? 

Cheers

**Comment by gunterze** [comment, marked as answer]
We are aware about it. Unfortunately the payment for the server lease was frozen by mistake. It may take several days until it get processed by the server provider and the server get reactivated :(

**Reply by darktyco** [reply]
Is there a backup mirror for the maven repo? Or any chance that the latest version can be published to maven central?

**Reply by gunterze** [reply]
It's up again.
